src/main/PDF/Services/V2Services/FillPDFServiceV2.java

```diff
@@ -211,7 +188,9 @@ public Message mergeFileAndFormQuestions(
     }
     try {
       setPDFFieldsFromFormQuestions(formQuestions, acroForm);
-      signPDF();
+      if (this.signatureStream != null) {
+        signPDF();
+      }
```

Comment:
if signature stream is null, we just return the unsigned pdf right

Reply:
Yea, that's why I added this line of code, as it was getting an error if the signature stream is null